crowbar_framework/app/models/service_object.rb

```diff
@@ -1713,4 +1714,34 @@ def active_update(proposal, inst, in_queue, bootstrap = false)
       [400, e2.message]
     end
   end
+
+  def skip_unready_nodes(bc, inst, elements)
+    skip_unready_nodes_config = Rails.application.config.experimental["skip_unready_nodes"]
```

Comment:
Could we possibly re-read the yaml config file here? I'd hate to have people restart crowbar just to change this setting.

Reply:
well, thats kind of the point of having this into a file instead of just a toggle. We could store that into the database as well so it would be easier, but I was thinking that when you need to go and restart crowbar, there is intention and plan it and follow up on the changes you did, so people dont just go, enable and forget. They understand that they are changing something and it requires a restart to enable, so they may be more careful this way.

All about that physiological barrier, maybe Im reading to much into this, but I know I think it twice and plan when having to restart something for a new option/config/feature VS just toggling something somewhere and forget.